R/opendp/src/opendp.h

```diff
@@ -1092,6 +1092,29 @@ struct FfiResult_____AnyMeasure opendp_measures__approximate(const struct AnyMea
  */
 struct FfiResult_____AnyMeasure opendp_measures__zero_concentrated_divergence(void);
 
+/**
```

Comment:
I think this is a generated file? Should it be gitignored? At least there is a `generate_Ropendp_h`.